_posts/2016-4-20-The Fashion Logging System -- ELK - Part 1.md

```diff
@@ -0,0 +1,161 @@
+# The Fashion Logging System -- ELK - Part 1
+
+## 前言：
+在做雲端服務的時候，有一個非常重要的環節，那就是 logging system！！  
+因為沒有 logging system 來幫你蒐集 log 並讓你容易 search 的話，先不論發生 bug 的時候不知道要從哪邊開始找起，就連伺服器掛掉狂噴 error 的時候，有可能客戶都會比你早發現，到時候就...  
+在威脅完大家 logging system 的重要性之後，這系列文章主要會分成以下三個部分去說明，讓大家對於 ELK 這套 logging system 有大致上的了解，並且可以架設出 ELK:
+1. ***什麼是 ELK ？***
+2. 用 EC2 架設 Logstash，並搭配 AWS 提供的 Elasticsearch 與 Kibana 來架構 ELK
+3. 將 Nginx 的 Log 藉由 Logstash 送往 AWS Elasticsearch，並用 Kibana 來監測 HTTP Status
+
+## 1. 什麼是 ELK ?
+ELK 分別是下面這三種東西
+- E: Elasticsearch
+- L: Logstash
+- K: Kibana
+
+首先我們會先從 Elasticsearch 說明起
+
+#### Elasticsearch
```

Comment:
This header (and further ones) could be promote up, becomes `## Elastisearch` for better readability.